react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class NewsletterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {email: ""};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label>
+          Email:
+          <input type="email" name="email" onChange={this.handleChange}></input>
```

Comment:
The `value` prop should be set from state on the input:

```JSX
<input type="email" name="email" onChange={this.handleChange} value={this.state.email} />
```

Also input tags are self closing so you don't need a closing tag